### PR TITLE
added JSON tsx for projects

### PR DESCRIPTION
**Pull request by 54skyxenon**
I made a TSX file `ProjectJSON.tsx` exporting a JSON array for project data in `src/pages/projects`, then used a map to render that in the original `Projects.tsx` source file.

Extras: Added a line to the README to indicate the command to lint. NPM warned me to bump the `react-scripts` version, so I did.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.<br />
 You will also see any lint errors in the console.
 
+### `npm run fix`
+
+Lints all of the TS source files. Do this ideally before you commit!
```
Comment (rymaju) [inline README.md:19]: ty

File: src/pages/projects/Projects.tsx [hunk 2]
```diff
@@ -23,32 +25,23 @@ const Projects: React.FC = () => {
...
-                C4C built a block reservation system, allowing SFFT's volunteers to reserve blocks of trees that they will \
-                be able to inventory and then mark them as complete when they are finished collecting the required data."}
-            appDesc={"Tree Inventory System"}
-            path="https://c4cneu-public.s3.us-east-2.amazonaws.com/Site/sfft-project-page.png"
-            abbr="SFFT"
-            learnMore="https://app.sfttbos.org" />
+          { 
+            ProjectJSON.cards.map((value, index) => {
+              return <ProjectCard
+                title={value.title}
```
Comment (rymaju) [inline src/pages/projects/Projects.tsx]: change this to take in props instead, as in:
` <ProjectCard {...value} />`

Comment (54skyxenon) [reply src/pages/projects/Projects.tsx]: fixed

**Review by rymaju — APPROVED** [review]
LGTM 👍

**Pull request merged**